src/redux/categories/categoriesSlice.js

```diff
@@ -11,4 +11,5 @@ const categoriesSlice = createSlice({
   reducers: {},
 });
 
+export const selectStatus = (state) => state.categories.status;
```

Comment:
- [ ] As suggested by the previous reviewer please add `checkstatus` reducer in the categorySlice and export it as an action of the slice. Example snippet:

```js
// ./src/redux/categories/categoriesSlice.js

import { createSlice } from "@reduxjs/toolkit"

export const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
})

export const { checkStatus } = categoriesSlice.actions

export default categoriesSlice.reducer

```